CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Changelog
+
+## 1.0.0 (April 24, 2017)
+
+### Features:
+
+* Use `cardano-sl` version `0.4.0`
```

Comment:
> Use `cardano-sl` version `0.4.0`

0.4.3 can have some critical bug fixes and I would advice to use that one. Also, its an imperative that we use the same version of cardano-sl that is deployed in production as @domenkozar and others already said once.